src/components/media/SearchResults.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement } from 'react';
+import * as ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+
+// Logic
+import { seasonYear } from '../../logic/dateFunctions';
+
+// GraphQL
+import { SearchMediaByTitleQuery } from './search_media.types';
+
+// Components
+import Title from '../../styles/components/Title';
+import { SubtypeTag } from '../../styles/components/Tag';
+import Search from 'src/styles/layouts/Search';
+import Poster from 'src/styles/components/Poster';
+
+interface Props {
+  data: SearchMediaByTitleQuery;
+}
+
+export default function SearchResults({ data }: Props): ReactElement {
```

Comment:
Can we arrow function this?

Reply:
Oh, right, I was using a snippet for setting it up and it uses classic functions by default. I still personally prefer them for functional components as it makes it much clearer where the boundaries are, but we can use arrow functions if you prefer that

Reply:
Fixed